src/openforms/config/views.py

```diff
@@ -95,6 +94,7 @@ def get_register_entries(self, register):
     def get_plugin_entry(self, plugin: Any) -> Entry:
         # undocumented query string support - helps for developers ;)
         requested_plugin = self.request.GET.get("plugin")
+        status_message, status, error = None, True, None
```

Comment:
```suggestion
        status_message, status, error = None, True, ""
```
-> matches type declaration for `Entry`